src/actions/actions.js

```diff
@@ -1,11 +1,42 @@
 import dataNbaNet from '../apis/dataNbaNet';
 
-export const getPlayers = () => async dispatch => {
-    const response = await dataNbaNet.get('/json/cms/noseason/team/raptors/roster.json');
+export const getTeams = () => async dispatch => {
+    const allTeamsResponse = await dataNbaNet.get("/prod/2019/teams_config.json");
+
+    const nbaTeamsArray = Object.values(allTeamsResponse.data.teams.config).filter(team => team.ttsName);
+
+    dispatch({ type: 'GET_TEAMS', payload: nbaTeamsArray });
+    
+    const raptorsTeam = nbaTeamsArray.find(team => team.teamId === "1610612761"); 
+
+    dispatch({ type: 'GET_SELECTED_TEAM', payload: raptorsTeam });
+
+    dispatch({ type: 'GET_TEAM_COLOR', payload: raptorsTeam.primaryColor });
+
+    const raptorsRosterResponse = await dataNbaNet.get("/json/cms/noseason/team/raptors/roster.json");
+
+    dispatch({ type: 'GET_PLAYERS', payload: raptorsRosterResponse.data.sports_content.roster.players.player });
+};
+
+export const getSelectedTeam = team => async dispatch => {
+    dispatch({ type: 'GET_SELECTED_TEAM', payload: team });
+
+    const teamUrlName = team.ttsName.trim().split(" ").pop().toLowerCase();
```

Comment:
There is a problem with one of the dropdown menu items. If you click on `Philadelphia 76ers` the data doesn't load. The reason being is that this `76ers` name does not relate to the `teamUrlName`. Instead they are actually referred to as the `sixers`.

In order to fix this bug, create a conditional to check if `team.ttsName` is `Philadelphia 76ers` then `teamUrlName` will be assigned `sixers`, otherwise do your trimming splitting etc.